### PR TITLE
NF-410 NF-411 NF-422 content changes to questions

### PR DESCRIPTION
**Pull request by rorticus0502**
content changes to questions based on RAD

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -29,10 +29,10 @@ service.unavailable.title = Sorry, the service is unavailable
 service.unavailable.paragraph.1 = We are working on updating this service.
 service.unavailable.paragraph.2 = In the meantime you can continue to submit NIDAC claims using the existing method.
 
-representation_type.title = Are you the importer or their representative?
-representation_type.error.required = Select if you are the importer or their representative
-representation_type.Importer = I am the importer
-representation_type.Representative_of_importer = I am a representative of the importer
+representation_type.title = Are you the importer?
+representation_type.error.required = Confirm if you are the importer or their representative
```
Comment (SteveSugden) [inline conf/messages:33]: OBSERVATION:  That should really be something like "Select yes if you are the importer" to conform to guidelines

**Review by SteveSugden — APPROVED** [review]

File: conf/messages (hunk 1 above)
Comment (rorticus0502) [reply conf/messages:33]: I asked for guidance from design and this this what they came up with

Comment (rorticus0502) [reply conf/messages:33]: I prefer your suggestion actually, it is more instructive

Comment (SteveSugden) [reply conf/messages:33]: Thanks.   FYI - https://design-system.service.gov.uk/components/radios/#error-messages

**Pull request merged**